ixmp/tests/test_tutorials.py

```diff
@@ -7,53 +7,73 @@
 
 from ixmp.testing import get_cell_output, run_notebook
 
+group_base_name = platform.system() + platform.python_version()
+
+GHA = "GITHUB_ACTIONS" in os.environ
+
+
 FLAKY = pytest.mark.flaky(
     reruns=5,
     rerun_delay=2,
-    condition="GITHUB_ACTIONS" in os.environ and platform.system() == "Darwin",
-    reason="Flaky; see iiasa/ixmp#489",
+    condition="GITHUB_ACTIONS" in os.environ and platform.system() == "Windows",
+    reason="Flaky; see iiasa/ixmp#543",
 )
 
 
+def default_args():
+    """Default arguments for :func:`.run_notebook."""
+    if GHA:
+        # Use a longer timeout
+        return dict(timeout=30)
+    else:
```

Comment:
Could add `else: # pragma: no cover` or collapse the whole function to a one-liner.

Reply:
Also, the way this function is used makes it seem like it could be a fixture. No?